Contributing/readme.md

```diff
@@ -122,17 +122,17 @@ Example component:
 </MyComponent>
 ```
 
-Example test utilizing testing-library utility:
+Example Jest test utilizing testing-library utility:
 
 ```js
 import { screen } from "@testing-library/react";
 
-it("Should display additional referrals count", () => {
-  cy.findByTestId("additional-referrals-count").contains("+ 2 more");
-});
+render(<MyComponent />);
+
+const element = screen.getByTestId("additional-referrals-count");
```

Comment:
Would it make sense to put this in a `test('...')` block to more closely resemble a real test?

```suggestion
it("Should display additional referrals count", () => {
  render(<MyComponent />);
  
  const element = screen.getByTestId("additional-referrals-count");
  // ...
});
```